_posts/2017-10-25-rails-girls-berlin-is-growing-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Rails Girls Berlin is growing up"
+date: "2017-10-25"
+coverImage: "IMG_0876b.jpg"
+---
+
+Rails Girls Berlin has existed since its first workshop in 2012. So far, we’ve organized more than 30 workshops with more than 1200 learners. From within our community, great projects such as [speakerinnen.org](http://speakerinnen.org), [rorganize.it](http://rorganize.it) and [diversitytickets.org](http://diversitytickets.org) have developed. Additionally, projects like [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/), [ClojureBridge Berlin](http://clojurebridge-berlin.org) and [Heart of Code](http://heartofcode.org) are partially built by people who found or refreshed their love for programming with us.
+
+ 
+
+Five years of Rails Girls Berlin also means that our organizers team has had quite a few generations - which is to be expected when it comes to unpaid volunteer work. However, there is one discussion that popped up for each of these generations: it’s about our name. As organizers we have always tried to not speak of “girls” but of learners, coaches, organizers. Nonetheless, it happens that people from our community are referred to as girls, for which you can hardly blame anyone if you just look at our name.
+
+ 
+
+In general, we’re not angry about it. Girls are great and we’re happy to rock to Beyoncé’s “Who run the world? - Girls!”. The playful approach, the let’s-just-try-and-see perspective to coding is probably also associated to not being an adult. Beyond that, we’ve had the feeling that this name has helped to make us appear less threatening - maybe a reason why we have received a rather small amount of attacks or backlashes compared to other initiatives for marginalized groups in tech. Thus, we were able to inspire people to code and question stereotypes about tech without much interference.
+
+ 
+
+Nonetheless, our initiative is aimed towards adults. Mainly it’s for women and non-binary people and eventually, in theory, we would even accept men. However, with roughly 200 applications for 45 spots, they are given the lowest priority. But that’s not reflected in our name. Instead, we have school approaching us, asking if we could do a workshop for kids or teenagers. Don’t get us wrong - early tech education is important, but with the little resources we have, we need to stick to one target group.
+
+ 
+
+“Girls” is perfectly fine as a self-chosen term - choose whichever term you feel comfortable with for yourself. However, we no longer want this term to be used as a general description for members of our community. So, we need to change the name!
+
+ 
+
+And since we’re already cleaning up, let’s be extra thorough: Rails is a great framework but we would like to keep it open for the future to maybe try a different framework or even programming languages. For now, that’s still very far away, but we could be prepared for it anyways.
```

Comment:
And since we’re already cleaning up, let’s dig even deeper. Ruby on Rails is a wonderful web framework, but we would like to keep keep our tool set open and explore different frameworks and languages e.g. Javascript, Rust and Elixir.